internal/audits/audits.go

```diff
@@ -0,0 +1,105 @@
+package audits
```

Comment:
I would consider sticking with the name capability. That package will be replaced by this one anyway as I'm understanding. I think capability identifies more precisely what that package is doing. Although I'm not against audit. Which brings me to another important note that would be avoiding using plural names to packages. Seems to be something adopted widely in the Go community.

Reply:
I can move it back. I did this in order to make it obvious where they were being called from. No problem with just moving it all back.

Reply:
We can totally change what was capAudit. This was the first sketch we could come up with. I agree that there are too many things tied together that need better separation.

Reply:
I've moved it back into capability. Definitely a more incremental change. It will make it easier to review.